### PR TITLE
Clean-up of near-zero coordinates in graphical annotations

### PR DESCRIPTION
**Pull request by DagBruck**
Minor clean-up of near-zero coordinates in graphical annotations.

### CODE REVIEW

File: Modelica/Electrical/Batteries/BaseClasses/BaseCellWithSensors.mo
Comment (tobolar) [inline Modelica/Electrical/Batteries/BaseClasses/BaseCellWithSensors.mo:69]: Double {0,-75},{0,-75} - can be squashed.

File: Modelica/Electrical/Batteries/Examples/SuperCapDischargeCharge.mo
Comment (tobolar) [inline Modelica/Electrical/Batteries/Examples/SuperCapDischargeCharge.mo:49]: Double {-12,0},{-12,0} - can be squashed.

File: Modelica/Electrical/Machines/Examples/SynchronousMachines/SMEE_DOL.mo
Comment (tobolar) [inline Modelica/Electrical/Machines/Examples/SynchronousMachines/SMEE_DOL.mo:167]: Double {0,60},{0,60} and {0,50},{0,50} - can be squashed.

File: Modelica/Electrical/PowerConverters/Examples/DCDC/ExampleTemplates/ChopperStepUp.mo
Comment (tobolar) [inline Modelica/Electrical/PowerConverters/Examples/DCDC/ExampleTemplates/ChopperStepUp.mo:67]: Clean-up not fully successful here ({1.77636e-15,60}).

File: Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo
Comment (tobolar) [inline Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo:44]: Double {-4,0},{-4,0} - can be squashed.

File: Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo
Comment (tobolar) [inline Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo:46]: Double {-34,0},{-34,0} - can be squashed.

File: Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo
Comment (tobolar) [inline Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo:154]: Double {0,10},{0,10} - can be squashed.

File: Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo
Comment (tobolar) [inline Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo:165]: Double {-46,0},{-46,0} - can be squashed.

File: Modelica/Electrical/PowerConverters/DCDC/Control/Voltage2DutyCycle.mo
Comment (tobolar) [inline Modelica/Electrical/PowerConverters/DCDC/Control/Voltage2DutyCycle.mo:47]: Double {0,80},{0,80} - can be squashed.

File: Modelica/Electrical/PowerConverters/DCDC/ChopperStepUp.mo
Comment (tobolar) [inline Modelica/Electrical/PowerConverters/DCDC/ChopperStepUp.mo:56]: Double {-16,0},{-16,0} - can be squashed.

File: Modelica/Electrical/Machines/Sensors/VoltageQuasiRMSSensor.mo
Comment (tobolar) [inline Modelica/Electrical/Machines/Sensors/VoltageQuasiRMSSensor.mo:43]: Double {0,-49.5},{0,-49.5}. It seems it can be deleted completely because the remaining {{0,-41},{0,-58}} makes the line!

**Review by hubertus65 — APPROVED** [review]

**Comment by TManikantan** [comment]
@DagBruck can you please make changes suggested by @tobolar . Many Thanks

**Comment by arunkumar-narasimhan** [comment]
@DagBruck, would you like our help in addressing @tobolar's comments? These are minor suggestions that we could also help close if you are stressed for time.

File: Modelica/Electrical/Batteries/Examples/SuperCapDischargeCharge.mo
Comment (tobolar) [reply Modelica/Electrical/Batteries/Examples/SuperCapDischargeCharge.mo]: Have commited this.

File: Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo
Comment (tobolar) [reply Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo]: Have commited this.

File: Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo
Comment (tobolar) [reply Modelica/Thermal/FluidHeatFlow/Examples/TestCylinder.mo]: Have commited this.

File: Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo
Comment (tobolar) [reply Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo]: Have commited this.

File: Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo
Comment (tobolar) [reply Modelica/Electrical/PowerConverters/Examples/ACAC/SoftStarter.mo]: Have commited this.

File: Modelica/Electrical/PowerConverters/DCDC/Control/Voltage2DutyCycle.mo
Comment (tobolar) [reply Modelica/Electrical/PowerConverters/DCDC/Control/Voltage2DutyCycle.mo]: Have commited this.

File: Modelica/Electrical/PowerConverters/DCDC/ChopperStepUp.mo
Comment (tobolar) [reply Modelica/Electrical/PowerConverters/DCDC/ChopperStepUp.mo]: Have commited this.

File: Modelica/Electrical/Machines/Sensors/VoltageQuasiRMSSensor.mo
Comment (tobolar) [reply Modelica/Electrical/Machines/Sensors/VoltageQuasiRMSSensor.mo]: Have commited this.

**Comment by tobolar** [comment]
> would you like our help in addressing @tobolar's comments? 

It's not too much effort to change the code on our own and close the PR. But the reason why not to do it is that the Dymola's command/script shall be improved to catch also doubled coordinates in annotations.

I have fixed and commented some comments of me occuring several times. So only comments significant for the Dymola script's change are left.

**Comment by arunkumar-narasimhan** [comment]
> > would you like our help in addressing @tobolar's comments?
> 
> It's not too much effort to change the code on our own and close the PR. But the reason why not to do it is that the Dymola's command/script shall be improved to catch also doubled coordinates in annotations.
> 
> I have fixed and commented some comments of me occuring several times. So only comments significant for the Dymola script's change are left.

Thanks for the commits, @tobolar. I didn't want to overstep assignee's work that too without at least notifying the assignee. Appreciate the info on Dymola command/script feature. We will probe for second review.

**Comment by TManikantan** [comment]
@tobolar @hubertus65 kindly review the fix. This was necessary as new commits were made. Many Thanks...

**Review by tobolar — COMMENTED** [review]
>  kindly review the fix

IMO, there are two options
1. fix my comments and merge or
2. wait for @DagBruck response.

**Comment by tobolar** [comment]
> I would suggest to just accept the current status of the pull request and merge it.

Ok, I will fix the remaining models and we can merge then.

File: Modelica/Electrical/Machines/Examples/SynchronousMachines/SMEE_DOL.mo
Comment (tobolar) [reply Modelica/Electrical/Machines/Examples/SynchronousMachines/SMEE_DOL.mo]: Have commited this.

File: Modelica/Electrical/Batteries/BaseClasses/BaseCellWithSensors.mo
Comment (tobolar) [reply Modelica/Electrical/Batteries/BaseClasses/BaseCellWithSensors.mo]: Have commited this.

**Comment by TManikantan** [comment]
thankyou @tobolar . I guess its you who have to again review it. @tobolar  @hubertus65 would you please approve the PR?

**Comment by MartinOtter** [comment]
Its not clear to me why "this commit cannot be built". Can someone help?

**Review by hubertus65 — APPROVED** [review]
Looks good, thanks!

**Comment by hubertus65** [comment]
It looks like merging is still blocked since github doesn't know that Dag is with Dassault and that there is a signed cla. I think that by now all actual code changes have been by @tobolar, so that it should be possible to back out Dag's commits and retry. Not sure if anybody here can override this issue. @MartinOtter, any chance of moving this forward?

**Comment by DagBruck** [comment]
> It looks like merging is still blocked since github doesn't know that Dag is with Dassault and that there is a signed cla. I think that by now all actual code changes have been by @tobolar, so that it should be possible to back out Dag's commits and retry. Not sure if anybody here can override this issue. @MartinOtter, any chance of moving this forward?

This is confusing but to me the fastest way forward was for me to sign the CLA electronically, even though it out of principle should not be needed. So far I have not seen any change in the status of this pull request.

**Pull request merged**
